server/db/dbManager.ts

```diff
@@ -53,22 +61,24 @@ export class PostManager {
 		if (pageSize * (pageNum - 1) > count) {
 			return [];
 		}
-		let sorted;
+		let sorted = this.model.find({ parent: undefined });
```

Comment:
Have you used `undefined` to be more explicit? Maybe, it is better to use a comment here?

Reply:
I used parent undefined to find the top-level posts, because only comments have a defined parent

Reply:
To be honest I just didn't realize that the only other things which parent could be is undefined, but in the next revision of getAllPosts this is not an issue